Redo Economy page (#23)
EssentialsX/Wiki: Economy.md

```yml
# The format of currency, excluding symbols. See currency-symbol-format-locale for symbol configuration.
#
# "#,##0.00" is how the majority of countries display currency.
#currency-format: "#,##0.00"

# Format currency symbols. Some locales use , and . interchangeably.
# Some formats do not display properly in-game due to faulty Minecraft font rendering.
#
# For 1.234,50 use de-DE
# For 1,234.50 use en-US
# For 1'234,50 use fr-ch
#currency-symbol-format-locale: en-US
```

### Permissions
Players need permissions to use economy commands. For example, you can grant them the `essentials.pay` permission to allow them to use `/pay`.

Permissions for other economy commands can be found on [the EssentialsX permissions list](https://essinfo.xeya.me/permissions.html) (community maintained list).

## Signs
Signs may be used to create shops where players can buy and sell items/blocks to and from the server in exchange for in-game money. Most economy commands can be used on signs, such as `[Buy]`, `[Sell]`, or `[Trade]`. You can find an in-depth tutorial at [the original Essentials wiki](https://wiki.mc-ess.net/wiki/Sign_Tutorial).

> ## For plugin developers 

Author: Would it be better if this was split into a separate page (as [#1 (comment)](https://github.com/EssentialsX/Wiki/issues/1#issuecomment-570790614) says)?

# EssentialsX Economy

> EssentialsX comes with a simple, easy-to-use economy system. The built in economy API will redirect to any other detected economy system such as vault. This means you can use EssentialsX Economy as a economy API instead of Vault if your plugin is dependent on EssentialsX.

Reviewer: ```suggestion
EssentialsX comes with a simple, easy-to-use economy system. The built-in economy API will redirect to any other detected economy system such as Vault. This means you can use EssentialsX Economy as a economy API instead of Vault if your plugin is dependent on EssentialsX.
```

IMO the economy API doesn't belong in the opening paragraph. The first paragraph should ideally explain EssentialsX's economy to server admins, but the economy API is only useful for plugin developers. We could have a separate section about Vault and plugin developers further down.

Author: Oh, I forgot to move that :P

## Commands

> There are a number of commands for use in-game, including but not limited to:

Reviewer: ```suggestion
There are a number of commands that players and admins can use in-game, including:
```

>    - /eco, /economy: Allows you to take, give, or reset a player's money.

Reviewer: ```suggestion
   - /eco, /economy: Allows admins to take, give, or reset a player's money.
```

   - /sell: Sells an item and amount at the price specified in the worth.yml.
   - /balance, /bal, /money: Displays the current balance of a player.
   - /baltop, /balancetop: Displays a list of the richest people.

>    - /pay: Pays a specified player from your balance.

Reviewer: It might be better to wrap the commands in backticks (`) for consistency with other EssentialsX documentation.

For example, `/baltop`, `/balancetop` can be written in Markdown as:

```markdown
`/baltop`, `/balancetop`
```

## Configuration

> There are a number of different options related to the internal economy system which can be found in the `config.yml`.
> Most of the config options are explained adequately in the comments:

Reviewer: ```suggestion
There are a number of different options related to EssentialsX's economy features which can be found in the `config.yml`.

Most of the config options are explained adequately in the comments:
```

```yml
# The format of currency, excluding symbols. See currency-symbol-format-locale for symbol configuration.
#
# "#,##0.00" is how the majority of countries display currency.
#currency-format: "#,##0.00"

# Format currency symbols. Some locales use , and . interchangeably.
# Some formats do not display properly in-game due to faulty Minecraft font rendering.
#
# For 1.234,50 use de-DE
# For 1,234.50 use en-US
# For 1'234,50 use fr-ch
#currency-symbol-format-locale: en-US
```

#### Permissions
Players need permissions to use economy commands. For example, you can grant them the `essentials.pay` permission to allow them to use `/pay`.

Permissions for other economy commands can be found on [the EssentialsX permissions list](https://essinfo.xeya.me/permissions.html) (community maintained list).

## Signs & Shops
Signs may be used to create shops where players can buy and sell items/blocks to and from the server in exchange for money. 

#### Signs
Most economy commands can be used on signs, such as [buy], [sell], or [trade]. You can find an in-depth tutorial at [the original Essentials wiki](https://wiki.mc-ess.net/wiki/Sign_Tutorial)

#### Shops

> WIP

Author: I actually don't know much about what these really are, I've just used signs lol

Reviewer: Servers can create "admin" shops with `[Buy]` and `[Sell]` signs as well as `/sell`, or players can create their own player-to-player shops with `[Trade]` signs. Those are the only shop systems built into EssX, so it might be worth dropping this section for now?

Author: Done.
